README.md

```diff
@@ -8,85 +8,95 @@
 
 <img src="/docs/screenshots/search-site.png?raw=true" alt="Search for your site">
+
 <img src="/docs/screenshots/accept-install.png?raw=true" alt="Install in your site">
 
 The plugin is now correctly installed and ready to index your site.
+
 <img src="/docs/screenshots/installed.png?raw=true" alt="View your site information.">
 
 ### Indexing
 
-Now that you've installed our plugin, your next deploy to Netlify will launch a crawl that will populate an Algolia index.
-You can manually launch a new deploy by opening your latest deploy in Netlify and clicking on "Retry deploy" > "Deploy site".
+Once you've installed the plugin, your next Netlify deploy will trigger a crawl and populate an Algolia index. You can manually trigger a new deploy in Netlify by clicking **Retry deploy > Deploy site** on any deploy.
+
+Once we receive a build hook, the Algolia Crawler processes your website asynchronously. This operation takes some time, resulting in a short delay between the first deploy and the associated crawl. Your site and your Algolia index will be out of sync during that delay.
+
+You can find information about your current crawler in the [Netlify deploy logs](https://docs.netlify.com/monitor-sites/logs/#deploy-log). We create one crawler (targeting one Algolia index) per Git branch, so that you can have, for example, a production index on `master` and development index on `develop`.
 
-Once we receive a build hook, our Crawler will process your website asynchronously, and this operation takes some time.
-In other words, there will be a delay between the first deploy on a branch and the Algolia index being ready for usage.
-You will find the information about your current Crawler in the Netlify deploy logs.
-We will create **one Crawler targeting one Algolia index per git branch**, so you can have a production index on `master` and development index on `develop` for example.
 <img src="/docs/screenshots/deploy-logs.png?raw=true" alt="Netlify deploy logs.">
 
-You can click on the URL to open the current Crawler dashboard to follow the progress of the crawl.
+You can click on the crawler URL to follow the progress of your crawl.
+
 <img src="/docs/screenshots/crawler-overview.png?raw=true" alt="Your Crawler running.">
 
-After the crawl is done you can go take a look at your Algolia index to see the records we extracted automatically.
-We apply a default relevance configuration by default, but you can fine tune it as you want in the index settings.
-Find out [what is inside your records](/docs/schema.md).
+Once the crawl is done, you can check your Algolia index, which contains the extracted records. We apply a standard relevance configuration by default, but you can fine-tune it as you want in the index settings.
+
+> Find out [what's inside your records](/docs/schema.md).
+
 <img src="/docs/screenshots/algolia-index.png?raw=true" alt="Your Algolia Index.">
 
-### Install the frontend bundle
+### Installing the front-end bundle
```

Comment:
https://twitter.com/sanketsahu/status/1309387236811526149

Reply:
https://twitter.com/DvBerzon/status/1309449110777458688 🙂

Reply:
you must be fun at parties ! 😏 😆

Reply:
![SX5MzFdz6Nh6g](https://user-images.githubusercontent.com/5370675/94661927-f9785780-0307-11eb-9f36-cf1ba0817215.gif)